en/20/06.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@PatrickAlphaC Just a note: we never did something like this, I mean each lesson requires them to enter an answer. We may want to move this to the introduction, something like https://cryptozombies.io/en/lesson/14/. I'm not sure yet, need to put some thought into this.

Reply:
Ok! Let me know what's best. It's a bit of a transitional slide for sure. We can drop the text into the top of the next lesson if that's better?

Reply:
@PatrickAlphaC We can probably do this also. I'll spin up a local server and try to figure out how to go about it.